app/src/main/java/com/example/finding_tory/InventoryAdapter.java

```diff
@@ -105,4 +105,4 @@ public void clearSelection() {
         selectedItems.clear();
         notifyDataSetChanged();
     }
-}
+}
```

Comment:
Files should end in a newline. I know android studio sometimes doesn't do that here you've changed it from having a newline to not having one. Please revert.
```suggestion
}

```